### ISSUE TITLE
Script Error

### ISSUE THREAD

**Issue by GLSmyth** [litera/jquery-transpose#1]
When I attempt to use this I get the error "items.eq(index).detach is not a function, pointing to the line:

newOrder = newOrder.add(items.eq(index).detach().data(keys.itemKey, index));

**Comment by litera** [comment]
Which jQuery version are you using?
According to [jQuery doc](http://api.jquery.com/detach/) this function was added in version 1.4 so if you're using 1.3.2 tough luck...

**Comment by GLSmyth** [comment]
That may be it, I am getting the code from Google via http://ajax.googleapis.com/ajax/libs/jquery/1.3/jquery.min.js

**Comment by litera** [comment]
Ok so that is the reason. You can see from the CDN code that it's version 1.3.2 and as documented on jQuery `detach()` was introduced in version 1.4.

I suppose this solves the issue.

> _Note_: Upgrading from 1.3.2 to 1.4 most likely won't affect your existing code because mostly new features were introduced in 1.4. Not all but mostly. I suggest you check [jQuery 1.4 change log](http://api.jquery.com/category/version/1.4) and see if anything might affect your existing code.
